lua/nvim-treesitter/configs.lua

```diff
@@ -4,76 +4,96 @@ local queries = require'nvim-treesitter.query'
 local parsers = require'nvim-treesitter.parsers'
 local utils = require'nvim-treesitter.utils'
 
--- @enable can be true or false
--- @disable is a list of languages, only relevant if enable is true
--- @keymaps list of user mappings for a given module if relevant
--- @is_supported function which, given a ft, will return true if the ft works on the module
 local config = {
-  modules = {
-    highlight = {
+  modules = {},
+  ensure_installed = nil
+}
+local user_setup_data = nil
+print('run')
```

Comment:
Thanks for answering me question. Now you can remove this.

Reply:
lol oops. Sorry about that.